src/api/Container.js

```diff
@@ -195,7 +200,7 @@ class Container extends React.Component<ContainerProps, ContainerState> {
                               },
                           )}
                           ref={(component) => { this.component = component; }}
-                          style={[style, { paddingTop: transitionPoint }]}
+                          style={[style, { paddingTop: transitionPoint - this.state.androidOffset }]}
```

Comment:
This actually does the same as it was before:
```suggestion
                          style={[style, { paddingTop: transitionPoint - this.getValue() }]}
```

as animatedOffset === this.animatedValue._value